src/adapters/Cornerstone/Segmentation_4X.js

```diff
@@ -1059,6 +1072,21 @@ function insertPixelDataPlanar(
             continue;
         }
 
+        const sourceImageMetadata = cornerstone.metaData.get(
```

Comment:
Using `cornerstone.metaData` can cause errors if `cornerstone` is not globally available (when using webpack, for instance). Should `metadataProvider` option used instead? There are 3 occurrences of `cornerstone.metaData` in this file on master branch. @pieper

Reply:
@rogerbramon nice catch, fix applied in https://github.com/dcmjs-org/dcmjs/pull/280. @pieper please have a look